new-player/src/main/java/net/newpipe/newplayer/ui/common/utils.kt

```diff
@@ -55,6 +56,38 @@ import net.newpipe.newplayer.R
 import net.newpipe.newplayer.uiModel.EmbeddedUiConfig
 import java.util.Locale
 
+/** Get the [Activity] from local context. Assumes the activity exists!
+ * @return the activity
+ * @throws NullPointerException if there is no Activity
+ */
+@Composable
+internal fun activity(): Activity
+    = LocalContext.current.findActivity()!!
+
+/** Call block with the [Activity] from current context, if there is an activity.
+ *
+ * @param default: the default value if there is no activity
+ * @param block: the block to call with the activity
+  */
+@Composable
+internal fun <T>activity(default: T, block: @Composable Activity.() -> T): T =
+    when (val a = LocalContext.current.findActivity()) {
+        null -> default
+        else -> block(a)
+    }
+
+
+@Composable
```

Comment:
Consider adding @hide here, so this function does not show up on the KDoc documentation.